.github/workflows/buf.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: bufbuild/buf-setup-action@v1
-      - uses: bufbuild/buf-breaking-action@v1.1.2
+#      - uses: bufbuild/buf-breaking-action@v1.1.2
```

Comment:
This is temporary and will be reverted after this change is merged and then sync'ed to the [BSR](https://buf.build/bufbuild/reflect/).